package.json

```diff
@@ -41,16 +41,19 @@
         "stylelint-rule-creator": "5.1.0"
     },
     "devDependencies": {
-        "@types/jest": "28.1.8",
-        "@types/stylelint": "13.13.3",
-        "jest": "29.0.1",
-        "postcss": "8.4.16",
+        "@types/jest": "29.5.12",
+        "@types/stylelint": "14.0.0",
+        "jest": "29.7.0",
+        "postcss": "8.4.33",
         "postcss-less": "6.0.0",
-        "stylelint": "14.11.0",
+        "stylelint": "15.11.0",
         "stylelint-jest-rule-tester": "2.1.7",
-        "virmator": "1.4.6"
+        "virmator": "11.3.1"
     },
     "optionalDependencies": {
-        "fsevents": "2.3.2"
+        "fsevents": "2.3.3"
+    },
+    "peerDependencies": {
+        "stylelint": "<16.0.0"
```

Comment:
Hello, thanks for the PR! Could you please reduce the list of changes down to just these few `peerDependencies` lines? (and the accompanying line changes in `package-lock.json` as well).

Reply:
Yeah and the other commit, so just keep a80f5ec74f250460a3c3879a947c92c9dc55aa8e.

Motivation is to make the smallest change to make the package a bit better, marking the stylelint peer dependency to prevent accidental installations with the wrong stylelint version.

Sometime (soon hopefully) I'll have time to go through and upgrade _everything_ and then it'll also be able to support stylelint v16

Reply:
@electrovir Thanks for clarifying! Splitted the PRs in three parts. The peer dependency one you requested can be found here:
https://github.com/electrovir/stylelint-plugin-import/pull/25

The gitignore is a must for anyone who wants to contribute from a mac:
https://github.com/electrovir/stylelint-plugin-import/pull/24